target/linux/mediatek/dts/mt7986a-acer-predator-w6.dts

```diff
@@ -186,8 +186,23 @@
 		reset-gpios = <&pio 6 GPIO_ACTIVE_LOW>;
 		reset-assert-us = <10000>;
 		reset-deassert-us = <10000>;
-		/* LED0: nc ; LED1: nc ; LED2: amber ; LED3: green */
-		mxl,led-config = <0x0 0x0 0x370 0x380>;
+
+		leds {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			led@2 {
```

Comment:
Kernel docs recommends the following pattern `"(^led-[0-9a-f]$|led)"`. Can we use it?

Reply:
Fixed.